neo/VM/ExecutionEngine.py

```diff
@@ -948,6 +944,9 @@ def StepOver(self):
     def VM_FAULT_and_report(self, id, *args):
         self._VMState |= VMState.FAULT
 
+        if settings.log_level != LOGGING_LEVEL_DEBUG:
```

Comment:
There is also a `logger.error()` output below. Are we sure we only want to output that error when debug logging is enabled? Or should this also be checking for `VMFault.THROW` and `VMFault.THROWIFNOT`? I think what you have is probably intended and proper (both for performance and for reducing log output), but wanted to raise this just in case.